composer.json

```diff
@@ -18,7 +18,7 @@
     "require-dev": {
         "symfony/config": "2.*",
         "phpunit/phpunit": "3.7.*",
```

Comment:
I think it's always best to have it as a dependency, since I don't want to rely on people or services to have the right version installed (I noticed there are subtle changes between versions at least).